.github/workflows/publish_to_pypi.yml

```diff
@@ -1,4 +1,4 @@
-name: Publish Python 🐍 distribution 📦 to PyPI and TestPyPI
+name: Publish cloudpickle 🥒 distribution 📦 to PyPI and TestPyPI
```

Comment:
This workflow doesn't publish Python but cloudpickle :)
Result of a quick copy paste.